### DISCUSSION TITLE
rAudio 20230202: Misaligned infos in playback window

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1222] [Fixed Issues]
Preliminary note: 
Problem could be related to the fix for issue [https://github.com/rern/rAudio-1/discussions/1203](url) ("1st entry in playlist does not scroll in playback window").

The following screenshots were performed on a smartphone with a 1440x2560 pixels resolution. Pictures reduced to 360x640.

**Case 1: artist/title/album infos require no scrolling**
Portrait: Looks good:
![1a](https://user-images.githubusercontent.com/25383513/216758445-68879753-4049-4e54-9ddb-abd3c8311655.jpg)
Landscape: Misaligned to the left:
![1b](https://user-images.githubusercontent.com/25383513/216758452-20804bd2-d549-4c3e-9dfd-ad15fc3ca4bb.jpg)

**Case 2: artist/title/album infos require scrolling**
Portrait: Clipping
![2a](https://user-images.githubusercontent.com/25383513/216758571-bcd0e0d7-61ce-4e7f-ba64-9dc853cca518.jpg)
Landscape: Misaligned to the left:
![2b](https://user-images.githubusercontent.com/25383513/216758701-a6c4eb15-db2a-44c1-9e27-ad45120f071f.jpg)

**Comment by rern** [comment]
@klassikmann,

Please try the UPDATE branch.

**Comment by klassikmann** [comment]
@rern 
Fixed in the UPDATE branch. 
Thank you
